source/draw/gpu/opengl/OpenGLSurface.ooc

```diff
@@ -28,13 +30,13 @@ OpenGLSurface: abstract class extends GpuSurface {
 	_unbind: abstract func
 	draw: override func (action: Func) {
 		this _bind()
-		this context _backend setViewport(this viewport)
+		this context backend setViewport(this viewport)
 		if (this opacity < 1.0f)
-			this context _backend blend(this opacity)
+			this context backend blend(this opacity)
 		else if (this blend)
-			this context _backend blend()
+			this context backend blend()
 		else
-			this context _backend enableBlend(false)
+			this context backend enableBlend(false)
```

Comment:
Exact comparisons with floating point values are not good practice in general but as long as people don't calculate their 1.0f constant with rounding errors, it should work without an epsilon offset because of how the exponent and mantissa works.